python/test/unit/language/test_tuple.py

```diff
@@ -8,7 +8,7 @@
 @triton.jit
 def _tuple_increment(values):
     for i in tl.static_range(len(values)):
-        values[i] = values[i] + 1
+        values = values[:i] + (values[i] + 1, ) + values[i + 1:]
```

Comment:
If the initial test was designed this way. We shouldn't modify just to make the interpreter pass

Reply:
Also I think interpreter related tests haven't been enabled.
You should add the `@pytest.mark.interpreter` decorator and `Makefile` to include `test_tuple.py` when testing the interpreter.

Reply:
Ok, I added that annotation on the tests that do pass unmodified.